app/controllers/oauth2_controller.rb

```diff
@@ -5,6 +5,8 @@
 # @!visibility private
 class Oauth2Controller < ApplicationController
 
+  force_ssl if: -> { RailsApiAuth.force_ssl }
```

Comment:
I think the lambda isn't needed here, could just do `force_ssl if: RailsApiAuth.force_ssl`